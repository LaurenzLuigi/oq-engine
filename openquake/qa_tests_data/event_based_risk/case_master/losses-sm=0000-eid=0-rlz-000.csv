event_tag,event_set,aid,business_interruption,contents,nonstructural,occupants:float32,structural,business_interruption_ins,contents_ins,nonstructural_ins,occupants_ins:float32,structural_ins
grp=00~ses=0001~src=1:5~rup=296-01,1,0,6.79575E+01,4.18547E+02,6.27820E+02,1.35915E-03,2.36563E+02,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,7.00000E-01
grp=00~ses=0001~src=1:5~rup=296-01,1,1,1.48587E+02,1.21789E+03,1.49825E+03,2.97174E-03,2.92197E+02,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,7.00000E-01
grp=00~ses=0001~src=1:5~rup=296-01,1,2,3.46470E+01,1.96033E+02,2.94050E+02,6.92939E-04,0.00000E+00,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,0.00000E+00
grp=00~ses=0001~src=1:5~rup=296-01,1,3,3.36214E+01,3.63711E+02,5.07701E+02,6.72428E-04,0.00000E+00,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,0.00000E+00
grp=00~ses=0001~src=1:5~rup=296-01,1,4,1.34583E+02,9.49172E+02,1.24146E+03,2.69166E-03,5.02352E+02,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,7.00000E-01
grp=00~ses=0001~src=1:5~rup=296-01,1,5,1.32398E+03,5.00000E+03,1.50000E+04,2.64796E-02,3.52458E+03,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,7.00000E-01
grp=00~ses=0001~src=1:5~rup=296-01,1,6,7.36467E+01,4.50326E+02,6.75489E+02,1.47293E-03,2.35089E+02,7.00000E-01,7.00000E-01,7.00000E-01,0.00000E+00,7.00000E-01
