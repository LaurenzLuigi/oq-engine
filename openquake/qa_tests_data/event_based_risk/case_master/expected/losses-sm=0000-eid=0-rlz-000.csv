event_tag,year,aid,business_interruption,contents,nonstructural,occupants:float32,structural
grp=01~ses=0001~src=2~rup=483-01,1,0,2.19361E+01,1.11041E+02,1.66562E+02,4.38722E-04,0.00000E+00
grp=01~ses=0001~src=2~rup=483-01,1,1,2.36587E+01,1.66636E+02,2.49954E+02,4.73173E-04,0.00000E+00
grp=01~ses=0001~src=2~rup=483-01,1,3,2.17764E+01,1.35838E+02,1.65123E+02,4.35528E-04,0.00000E+00
grp=01~ses=0001~src=2~rup=483-01,1,4,3.69903E+01,2.10017E+02,3.15026E+02,7.39806E-04,0.00000E+00
grp=01~ses=0001~src=2~rup=483-01,1,5,2.38116E+01,1.25857E+02,1.88785E+02,4.76233E-04,0.00000E+00
grp=01~ses=0001~src=2~rup=483-01,1,6,2.12090E+01,1.07083E+02,1.60625E+02,4.24179E-04,0.00000E+00
